src/main/java/org/opentripplanner/raptor/rangeraptor/transit/AccessPaths.java

```diff
@@ -47,26 +110,50 @@ public int calculateMaxNumberOfRides() {
     );
   }
 
+  public IntIterator iterateOverPathsWithPenalty(final int earliestDepartureTime) {
```

Comment:
Can this be private? If not, can you add Javadoc?

Reply:
No, it is used in the main loop, looping over the raptor search-window. I added JavaDoc so it is more clear what this method is used for.